modules/client-common/src/main/java/org/apache/ignite/internal/client/proto/ClientMessagePacker.java

```diff
@@ -325,6 +326,33 @@ public ClientMessagePacker packUuid(UUID val) {
         return this;
     }
 
+    /**
+     * Writes an {@link IgniteUuid}.
+     *
+     * @param val {@link IgniteUuid} value.
+     * @return This instance.
+     */
+    public ClientMessagePacker packIgniteUuid(IgniteUuid val) {
```

Comment:
and unpackIgniteUuid need to be utility methods, i hope.

Reply:
If you mean that there should be an unpack counterpart, there is some org.apache.ignite.internal.client.proto.ClientMessageUnpacker#unpackIgniteUuid

Reply:
no i mean that it would be helpful to have a public static ByteBuffer packIgniteUuid(IgniteUuid val) method. Or it can`t be reused from some other places ?

Reply:
Well I believe that it's possible to use different approach with IgniteUUID in compassion to all other packing/unpacking logic in ClientMessagePacker and it's unpacker counterpart, however it'll be confusing.
Current IgniteUUID packing approach is similar to UUID approach in ClientMessagePacker that internally uses org.msgpack.core.MessagePacker, packer close state, etc. Besides that there is a ticket for enhancement - https://issues.apache.org/jira/browse/IGNITE-15234
As a result I believe that using utility method for IgniteUuid is out of the scope of Ignite-15404 ticket.